.dockerignore

```diff
@@ -1,5 +1,5 @@
-.*
-env
-geckodriver*
-Vagrantfile
-Dockerfile
+.*

```

Comment:
You've changed a lot of files' line endings to CRLF, which you generally don't want, plus it has made this review a little unwieldy.